lib/core/src/lib/notifications/components/notification-history.component.spec.ts

```diff
@@ -32,14 +33,17 @@ describe('Notification History Component', () => {
     let notificationService: NotificationService;
     let overlayContainerElement: HTMLElement;
     let storage: StorageService;
+    let testNotifications: NotificationModel[];
 
     const openNotification = () => {
         fixture.detectChanges();
         const button = element.querySelector<HTMLButtonElement>('#adf-notification-history-open-button');
-        button.click();
+        button?.click();
```

Comment:
this might mean you are hiding a problem with button not available

Reply:
This is not an attempt to hide anything. the '?' was added to remove red line errors which are currently a part of the develop branch.
![Screenshot from 2023-11-02 17-25-04](https://github.com/Alfresco/alfresco-ng2-components/assets/110394264/886aa476-a72f-4a9c-8f24-2b1b6b3ceab0)
![Screenshot from 2023-11-02 17-25-20](https://github.com/Alfresco/alfresco-ng2-components/assets/110394264/1b75ffea-a80c-48f8-99b7-f755c810669b)
![Screenshot from 2023-11-02 17-25-34](https://github.com/Alfresco/alfresco-ng2-components/assets/110394264/ac5bc362-1f37-43a3-814d-eff84689fc4a)





I added '?' [here](https://github.com/Alfresco/alfresco-ng2-components/pull/9011/files#diff-0d99474bfab4c4b5265ff61b8fb06921dc9f985328a03c7a0b20aedda1283708R127) and [here](https://github.com/Alfresco/alfresco-ng2-components/pull/9011/files#diff-0d99474bfab4c4b5265ff61b8fb06921dc9f985328a03c7a0b20aedda1283708R149) for the same reason.

The tests work fine without the '?' as well but give a red line in the code. If you suggest, I will revert these.